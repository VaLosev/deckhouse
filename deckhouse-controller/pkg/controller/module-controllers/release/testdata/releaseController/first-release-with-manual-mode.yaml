apiVersion: deckhouse.io/v1alpha1
kind: ModuleUpdatePolicy
metadata:
  name: parca
spec:
  moduleReleaseSelector:
    labelSelector:
      matchLabels:
        source: parca
  releaseChannel: Stable
---
apiVersion: deckhouse.io/v1alpha1
kind: ModuleRelease
metadata:
  name: parca-1.25.1
  labels:
    module: parca
    modules.deckhouse.io/update-policy: parca
spec:
  moduleName: parca
  version: 1.25.1
  weight: 900
status:
  approved: false
  phase: Pending
  transitionTime: "2021-11-08T08:34:01Z"
