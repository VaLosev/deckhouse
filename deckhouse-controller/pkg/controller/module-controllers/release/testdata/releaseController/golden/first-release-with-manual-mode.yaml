---
apiVersion: deckhouse.io/v1alpha1
kind: ModuleRelease
metadata:
  creationTimestamp: null
  labels:
    module: parca
    modules.deckhouse.io/update-policy: parca
  name: parca-1.25.1
  resourceVersion: "1000"
spec:
  moduleName: parca
  version: 1.25.1
  weight: 900
status:
  approved: false
  message: 'Release is waiting for the ''modules.deckhouse.io/approved: "true"'' annotation'
  phase: Pending
  pullDuration: 0s
  size: 0
  transitionTime: "2019-10-17T15:33:00Z"
